xds/src/main/java/io/grpc/xds/LocalityStore.java

```diff
@@ -497,5 +443,177 @@ protected Helper delegate() {
         return orcaReportingHelperWrapper.asHelper();
       }
     }
+
+    private final class PriorityManager {
+
+      private static final int INVALID_PRIORITY = -1;
```

Comment:
You don't really need this, `-1` suffices, as you are not exporting this information outside.

Reply:
Done.